core/src/main/java/org/hibernate/ogm/type/descriptor/impl/Iso8601CalendarTypeDescriptor.java

```diff
@@ -103,12 +105,19 @@ public Calendar fromString(String string) {
 
 			@Override
 			public String toString(Calendar value) {
+				clearCalendar( value );
 				return DatatypeConverter.printDateTime( value );
 			}
 		};
 
 		public abstract Calendar fromString(String string);
 
 		public abstract String toString(Calendar value);
+
+		protected void clearCalendar(Calendar value) {
```

Comment:
Any particular reason to change this test? It makes the build fails and it changes the expected behaviour. Also, why should we remove the milliseconds?